float64 x
float64 y
float64 z
float64 yaw

uint8 game_progress   
uint16 stage_remain_time     
uint16 bullet_remaining_num_17mm    
uint32 Seat

float64 test